openmetadata-ui/src/main/resources/ui/playwright/utils/lineage.ts

```diff
@@ -0,0 +1,386 @@
+  await page.click('[data-testid="lineage-layer-btn"]');
+  await page.click('[data-testid="lineage-layer-column-btn"]');
+};
+
+export const performZoomOut = async (page: Page) => {
+  for (let i = 0; i < 5; i++) {
+    const zoomOutBtn = page.locator('.react-flow__controls-zoomout');
+    const enabled = await zoomOutBtn.isEnabled();
+    if (enabled) {
+      zoomOutBtn.dispatchEvent('click');
+    }
+  }
+};
+
+export const deleteEdge = async (
+  page: Page,
+  fromNode: EntityClass,
+  toNode: EntityClass
+) => {
+  const fromNodeFqn = get(fromNode, 'entityResponseData.fullyQualifiedName');
+  const toNodeFqn = get(toNode, 'entityResponseData.fullyQualifiedName');
+
+  page.click(`[data-testid="edge-${fromNodeFqn}-${toNodeFqn}"]`, {
+    force: true,
+  });
+
+  if (
+    ['Table', 'Topic'].indexOf(fromNode.getType()) > -1 &&
+    ['Table', 'Topic'].indexOf(toNode.getType()) > -1
+  ) {
+    await page.locator('[data-testid="add-pipeline"]').dispatchEvent('click');
+    await page
+      .locator(
+        '[data-testid="add-edge-modal"] [data-testid="remove-edge-button"]'
+      )
+      .dispatchEvent('click');
+  } else {
+    await page.locator('[data-testid="delete-button"]').dispatchEvent('click');
+  }
+
+  await expect(page.locator('[role="dialog"]')).toBeVisible();
+
+  const deleteRes = page.waitForResponse('/api/v1/lineage/**');
+  await page
+    .locator('[data-testid="delete-edge-confirmation-modal"] .ant-btn-primary')
+    .dispatchEvent('click');
+  await deleteRes;
+};
+
+export const dragConnection = async (
+  page: Page,
+  sourceId: string,
+  targetId: string,
+  isColumnLineage = false
+) => {
+  const selector = !isColumnLineage
+    ? '.lineage-node-handle'
+    : '.lineage-column-node-handle';
+
+  const lineageRes = page.waitForResponse('/api/v1/lineage');
+  await page
+    .locator(`[data-testid="${sourceId}"] ${selector}.react-flow__handle-right`)
+    .dispatchEvent('click');
+  await page
+    .locator(`[data-testid="${targetId}"] ${selector}.react-flow__handle-left`)
+    .dispatchEvent('click');
+
+  await lineageRes;
+};
+
+export const connectEdgeBetweenNodes = async (
+  page: Page,
+  fromNode: EntityClass,
+  toNode: EntityClass
+) => {
+  const type = getEntityTypeSearchIndexMapping(toNode.type);
+  const fromNodeFqn = get(fromNode, 'entityResponseData.fullyQualifiedName');
+  const toNodeName = get(toNode, 'entityResponseData.name');
+  const toNodeFqn = get(toNode, 'entityResponseData.fullyQualifiedName');
+
+  await page.locator(`[data-testid="${type}-draggable-icon"]`).hover();
+  await page.mouse.down();
+  await page.locator('[data-testid="lineage-details"]').hover();
+  await page.mouse.up();
+
+  await page.locator('[data-testid="suggestion-node"]').dispatchEvent('click');
+
+  const suggestionNodeInput = page.locator(
+    '[data-testid="suggestion-node"] input'
+  );
+  await suggestionNodeInput.dispatchEvent('click');
+  await suggestionNodeInput.fill(toNodeName);
```

Comment:
We can directly get element and type in it
```suggestion
 await page.locator(
    '[data-testid="suggestion-node"] input'
  ).fill(toNodeName);
```
or
```suggestion
 await page.fill(
    '[data-testid="suggestion-node"] input', toNodeName
  );
```